src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -231,8 +229,11 @@ public AgentSettings LoadSettings()
                                                 args,
                                                 enforceSupplied);
 
-                // TODO: Scan for capabilites
-                var capabilities = new Dictionary<string, string>();
+                Dictionary<string, string> capabilities;
+                using (var ct = new CancellationTokenSource())
```

Comment:
`ct` isn't exposed outside the scope of the using and there is no code in the using that could possibly cancel it. CancellationToken.None would be more intuitive.